### ISSUE TITLE
Wheel inside a `<dialog>` disappears when resizing the window while the `<dialog>` is closed

### ISSUE THREAD

**Issue by matronator** [CrazyTim/spin-wheel#42]
I have put the wheel inside a `<dialog>` tag and want it to be visible only when the user clicks a button to show the dialog. However when the dialog is closed, if I resize the window and then open the dialog, the wheel is gone. I have to resize the window again for it to become visible.

I think it might be because the canvas is auto-adjusting its size on resize event, but when the dialog is hidden, it sets its size to 0px by 0px, thus disappearing and only after resizing again with the modal open it sets the correct dimensions for its size.

**Comment by matronator** [comment]
Okay, after looking at the source code, I found `resize()` method and if I call that right after I open the dialog, it shows the wheel again. But I didn't find any mention of this method anywhere, so maybe put it in the `README`?

**Comment by CrazyTim** [comment]
Duplicate of #26
